src/modules/commander/commander.cpp

```diff
@@ -2736,6 +2734,10 @@ int commander_thread_main(int argc, char *argv[])
 				(status.is_rotary_wing || (!status.is_rotary_wing && land_detector.landed)) &&
 				(stick_in_lower_left || (arm_button_pressed && arming_button_switch_allowed) || arm_switch_to_disarm_transition) ) {
 
+				/* counter for having stick in disarming position */
+				static unsigned stick_off_counter = 0;
```

Comment:
Why did you replace them with static variables? I would not do that, in my opinion this makes it a lot less clear with declaration, initialization and so on.

Reply:
it limits the scope to this part of the code. the commander already has a lot of global (modular scope) variables.